.gitignore

```diff
@@ -1 +1,3 @@
 .idea/
+
+vendor/
```

Comment:
We many not want to exclude the vendor directory here, as some Go users may prefer to use vendoring.